File: Modelo/PerfilPot.java
package Modelo;

public class PerfilPot {

	private int id;
	public float[] potencia = new float[10];
	
	public PerfilPot() {}

	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}

	public float[] getPotencia() {
		return potencia;
	}

	public void setPotencia(float[] potencia) {
		this.potencia = potencia;
	}
	
	
}

File: Modelo/PerfilPot.php
<?php
class PerfilPot{
/*  Variables */

private $id;
public $potencia = [10];

/* Constructor */
public function __construct(){}
/* Getters y setters */
public function getId(){
    return $this ->id;
}
public function setId($id){
    $this ->id = $id;
}
public function getPotencia(){
    return $this ->potencia;
}
public function setPotencia($potencia){
    $this ->potencia = $potencia;
}
}
?>